CONTRIBUTING.md

```diff
@@ -88,7 +93,12 @@ for version in 4.0 4.1 4.2 master; do
   else
     # red in ANSI
     echo -e "\033[31m **** Tests failed against Rails ${RAILS_VERSION} **** \033[0m"
-  fi
+    read -p '[Enter] any key to continue, [q] to quit...' prompt
+    if [ "$prompt" = 'q' ]; then
+      unset RAILS_VERSION
+      exit 1
+    fi
```

Comment:
I thought about this, but wasnt sure about a script floating around a gem, so I just left it in the docs